scripts/provision-demo-course.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Experimental! Not ready for production use.
+# Adapted from https://github.com/edx/configuration/tree/master/playbooks/roles/demo
```

Comment:
This script (and its LMS companion, provision-demo-users.sh) replaces the Ansible playbook in edx/configuration which provisions demo course and users for devstack.

I wasn't sure whether it belonged in here or in devstack. I chose here, in the spirit of "every service repo should have the materials necessary to build, run, and test itself", but I could be swayed into moving this code into edx/devstack.